packages/react/package.json

```diff
@@ -43,7 +43,8 @@
     "lint": "eslint ./src",
     "test:types": "tsc --noEmit --emitDeclarationOnly false",
     "test:unit": "vitest run",
-    "test:unit:watch": "vitest"
+    "test:unit:watch": "vitest",
+    "next": "node ./yalc.next.js && pnpm build && yalc publish"
```

Comment:
I'd think of a slightly more meaningful name for it, `next` makes me think of Next.js. How about `next-version` or `prerelease` ?

Reply:
or even `yalc` / `yalc-publish`

Reply:
Following the conventions we had so far, my vote would be `publish:local` or `publish:yalc`

Reply:
I renamed it `publish:yalc`